jbmc/unit/java_bytecode/java_bytecode_convert_method/convert_method.cpp

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************\
 
- Module: Unit tests for converting constructors and static initializers
+ Module: Unit tests for classes with bridge methods and native methods
```

Comment:
When this is rebased on develop, it will include tests for methods which are neither bridge, static, or native methods. Maybe find a more general formulation? `Unit tests for converting methods`?

Reply:
done